src/Symfony/Component/Notifier/Bridge/Novu/NovuSubscriberRecipient.php

```diff
@@ -63,4 +64,9 @@ public function getLocale(): ?string
     {
         return $this->locale;
     }
+
+    public function getOverrides(): array
```

Comment:
Why do we need a getter? I don't see any example of usage in your code

Reply:
@stloyd  see the readme of the component, you use it in the `NovuNotification` class